src/bbtag/subtags/bot/args.ts

```diff
@@ -52,7 +52,7 @@ export class ArgsSubtag extends CompiledSubtag {
             throw new NotANumberError(index);
 
         if (context.input.length <= i || i < 0)
```

Comment:
Would it be possible that `i` is
1) negative?
2) how would changing the `i` value to `i+1` in the error allow the user to understand the problem?

Reply:
yep, I could be negative, theres a test for that. This is just for the debug message, garbage in garbage out imo. The error message shown for code purposes is just `Not Enough Arguments`, it makes no mention of counts